.ci_support/osx64.yaml

```diff
@@ -8,3 +8,11 @@ channel_sources:
   - conda-forge
 target_platform:
   - osx-64
+
```

Comment:
Can you explain why you are modifying this file? Typically adding a package should not modify files outside of of the package's recipe directory.

Reply:
Not intended.  This seems to have gotten added when I was trying to build locally.  I have tried to remove this change.